firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -314,3 +315,64 @@ uint8_t TCMD_agenda_fetch(){
 
     return 0;
 }
+
+
+/// @brief Deletes a telecommand from the agenda by the telecommand function name.
+/// @param function_name The `timestamp_sent` value of the telecommand to delete.
+/// @return 0 on success, > 0 if the telecommand was not found.
+/// @note Calls `LOG_message()` to log the deletion before all returns.
+uint8_t TCMD_agenda_delete_by_function_name(const char *function_name) {
+    bool is_valid_function_name_count = false;
```

Comment:
Pease move declaration to right before the variable is used.

Reply:
Done. Moved it to above the for loop and I changed the name to `is_valid_telecommand_name`. The name sounds more intuitive.